packages/circuit-ui/package.json

```diff
@@ -75,6 +75,10 @@
     "react": ">=18.0.0 <19.0.0",
     "react-dom": ">=18.0.0 <19.0.0"
   },
+  "optionalDependencies": {
```

Comment:
TIL 👀

Reply:
Oh there are more

<img width="259" alt="Screenshot 2023-05-12 at 10 45 43" src="https://github.com/sumup-oss/circuit-ui/assets/35560568/43a3be5d-f641-4fb8-97ed-3a9ecfca15b7">

https://docs.npmjs.com/cli/v9/configuring-npm/package-json#dependencies

npm is broken